test/integration/targets/lineinfile/tasks/main.yml

```diff
@@ -717,37 +717,3 @@
   assert:
     that:
       - result.stat.checksum == 'eca8d8ea089d4ea57a3b87d4091599ca8b60dfd2'
```

Comment:
Keeping tests (but updating if the results have changed) seems like a good idea.  Then it's an additional clue that the behaviour was meant to be this way.